package.json

```diff
@@ -28,6 +28,7 @@
     "cors": "^2.7.1",
     "express": "^4.14.0",
     "jsonfile": "^2.3.1",
+    "moment": "^2.14.1",
```

Comment:
Please rebase the commit.. This file has been modified with additional dependencies.